voice-search/voice-search-impl/lint-baseline.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<issues format="6" by="lint 7.0.4" type="baseline" client="gradle" name="AGP (7.0.4)" variant="all" version="7.0.4">
```

Comment:
I didn't even know of these issues.  Probably didn't notice them.  😅 So this is really helpful.

Reply:
Yeah, not sure about the overdrawn ones but the unused resources is probably because they are used on a different module?